androidApp/src/androidTest/java/com/mbta/tid/mbta_app/android/stopDetails/AlertCardTests.kt

```diff
@@ -0,0 +1,108 @@
+package com.mbta.tid.mbta_app.android.stopDetails
+
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.test.assertIsDisplayed
+import androidx.compose.ui.test.assertIsNotDisplayed
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.compose.ui.test.onNodeWithText
+import androidx.compose.ui.test.performClick
+import com.mbta.tid.mbta_app.android.util.fromHex
+import com.mbta.tid.mbta_app.model.Alert
+import com.mbta.tid.mbta_app.model.ObjectCollectionBuilder
+import kotlin.test.assertTrue
+import org.junit.Rule
+import org.junit.Test
+
+class AlertCardTests {
```

Comment:
suggestion: I think it would be worth copying the integration tests from `StopDetailsFilteredDepartureDetailsTests` as well that cover alerts cases
* testShowsSuspension
* testShowsDownstreamAlert
* testShowsElevatorAlert